molgenis-app/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.1"
 services:
     nginx:
-      image: nginx:stable
+      image: molgenis/nginx:stable
```

Comment:
is this a different image ? , or the same from other repo ?

Reply:
This is our own located on hub.docker.com on the molgenis organisation. The configuration is located in the molgenis-frontend repository. The version needs to be updated.